colossalai/zero/sharded_optim/low_level_optim.py

```diff
@@ -35,7 +35,7 @@ def __init__(
             optimizer: Optimizer,
 
     # grad scaler config
-            initial_scale=2**32,
+            initial_scale=2**16,
```

Comment:
为什么改这个值？

Reply:
Because 2**32 is too large. [Nvidia Apex](https://github.com/NVIDIA/apex/blob/91fcaaf718306e3441e9a8369aa4553eea2e326b/apex/amp/scaler.py#L40) uses 2**16 here.